src/components/application_manager/test/commands/mobile/show_constant_tbt_test.cc

```diff
@@ -0,0 +1,381 @@
+  }
+
+  MessageSharedPtr CreateExpectedMessage() {
+    MessageSharedPtr msg = CreateMessage(smart_objects::SmartType_Map);
+
+    smart_objects::SmartObject msg_params =
+        smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+    int32_t index = 0;
+
+    msg_params[am::hmi_request::navi_texts][index]
+              [am::hmi_request::field_name] = static_cast<int32_t>(
+                  hmi_apis::Common_TextFieldName::navigationText1);
+    msg_params[am::hmi_request::navi_texts][index++]
+              [am::hmi_request::field_text] = kNavigationText1;
+
+    msg_params[am::hmi_request::navi_texts][index]
+              [am::hmi_request::field_name] = static_cast<int32_t>(
+                  hmi_apis::Common_TextFieldName::navigationText2);
+    msg_params[am::hmi_request::navi_texts][index++]
+              [am::hmi_request::field_text] = kNavigationText2;
+
+    msg_params[am::hmi_request::navi_texts][index]
+              [am::hmi_request::field_name] =
+                  static_cast<int32_t>(hmi_apis::Common_TextFieldName::ETA);
+    msg_params[am::hmi_request::navi_texts][index++]
+              [am::hmi_request::field_text] = kEta;
+
+    msg_params
+        [am::hmi_request::navi_texts][index][am::hmi_request::field_name] =
+            static_cast<int32_t>(hmi_apis::Common_TextFieldName::totalDistance);
+    msg_params[am::hmi_request::navi_texts][index++]
+              [am::hmi_request::field_text] = kTotalDistance;
+
+    msg_params[am::hmi_request::navi_texts][index]
+              [am::hmi_request::field_name] = static_cast<int32_t>(
+                  hmi_apis::Common_TextFieldName::timeToDestination);
+    msg_params[am::hmi_request::navi_texts][index++]
+              [am::hmi_request::field_text] = kTimeToDestination;
+
+    (*msg)[am::strings::msg_params] = msg_params;
+
+    return msg;
+  }
+
+  void ExpectedManageMobileCommandResultCode(mobile_apis::Result::eType code) {
+    EXPECT_CALL(app_mngr_, ManageMobileCommand(MobileResultCodeIs(code), _))
+        .WillOnce(Return(true));
+  }
+
+ private:
+  void SetUp() OVERRIDE {
+    message_ = CreateFullParamsUISO();
+    request_ = CreateCommand<ShowConstantTBTRequest>(message_);
+    expected_message_ = CreateExpectedMessage();
+    mock_app_ = CreateMockApp();
+    ON_CALL(app_mngr_, application(kConnectionKey))
+        .WillByDefault(Return(mock_app_));
+    ON_CALL(app_mngr_, GetPolicyHandler())
+        .WillByDefault(ReturnRef(mock_polici_handler_interface_));
+  }
+
+ protected:
+  MessageSharedPtr message_;
+  SharedPtr<ShowConstantTBTRequest> request_;
+  MessageSharedPtr expected_message_;
+  MockAppPtr mock_app_;
+  MockPolicyHandlerInterface mock_polici_handler_interface_;
+};
+
+TEST_F(ShowConstantTBTRequestTest, Run_AllCorrectData_SendHMIRequest) {
+  EXPECT_CALL(mock_message_helper_, ProcessSoftButtons(_, _, _, _))
+      .WillOnce(Return(mobile_apis::Result::SUCCESS));
+
+  EXPECT_CALL(mock_message_helper_, VerifyImage(_, _, _))
```

Comment:
@BSolonenko I think you can specify first two params of VerifyImage() call explicitly

Reply:
@AKalinich-Luxoft Fixed in https://github.com/smartdevicelink/sdl_core/pull/1977/commits/2127b6028b54933a3f410a26f186212ea4845467